bfdd/bfd.c

```diff
@@ -1030,7 +1030,16 @@ static void bs_init_handler(struct bfd_session *bs, int nstate)
 		break;
 
 	case PTM_BFD_DOWN:
-		/* Remote peer hasn't moved to first stage yet. */
```

Comment:
I'm not sure that this is the right fix.
I quickly looked at the code and I don't see that we start sending control packets for passive session anywhere, hence your problem.
To fix this correctly, I think we need to start sending control packets periodically once we receive the first packet from our peer. What do you think?

Reply:
Thanks！
Yes. It should use existed timer.